fabtests/pytest/efa/test_runt.py

```diff
@@ -59,9 +59,10 @@ def test_runt_read_functional(cmdline_args):
     client_send_bytes = client_send_bytes_after_test - client_send_bytes_before_test
 
     # Among the 256 KB of data, 64 KB data will be sent via 8 RUNTREAD RTM packets.
-    # Each packet has a packet header, therefore the total number of send bytes
-    # is slightly larger than 64 K.
-    assert client_send_bytes > 65536 and client_send_bytes < 66560
```

Comment:
So this assert is still correct for multi-node? If we remove the second assert, will it ignore some errors on multi-node?

Reply:
Thanks!